llpc/context/llpcCompiler.cpp

```diff
@@ -1609,7 +1644,12 @@ Result Compiler::buildComputePipelineInternal(ComputeContext *computeContext,
     result = buildPipelineWithRelocatableElf(context, shadersInfo, pipelineElf, stageCacheAccesses);
     *stageCacheAccess = stageCacheAccesses[ShaderStageCompute];
   } else {
+    const MetroHash::Hash originalPipelineHash = computeContext->getPipelineHashCodeWithoutCompact();
+    MetroHash::Hash cacheHash = {};
+    cacheHash = PipelineDumper::generateHashForComputePipeline(pipelineInfo, true, true);
+    context->getPipelineContext()->setPipelineHashCode(convertToHashUsedForCacheLookup(cacheHash));
```

Comment:
As before, this does not have to be done here.

Reply:
done